pkg/monitors/uptimerobot/uptime-monitor.go

```diff
@@ -86,14 +86,23 @@ func (monitor *UpTimeMonitorService) Add(m models.Monitor) {
 
 	client := http.CreateHttpClient(monitor.url + action)
 
-	body := "api_key=" + monitor.apiKey + "&format=json&type=1&url=" + url.QueryEscape(m.URL) + "&friendly_name=" + url.QueryEscape(m.Name) + "&alert_contacts=" + monitor.alertContacts
+	body := "api_key=" + monitor.apiKey + "&format=json&url=" + url.QueryEscape(m.URL) + "&friendly_name=" + url.QueryEscape(m.Name) + "&alert_contacts=" + monitor.alertContacts
 
 	if val, ok := m.Annotations["uptimerobot.monitor.stakater.com/interval"]; ok {
 		body += "&interval=" + val
 	}
 	if val, ok := m.Annotations["uptimerobot.monitor.stakater.com/maintenance-windows"]; ok {
 		body += "&mwindows=" + val
 	}
+	if val, ok := m.Annotations["uptimerobot.monitor.stakater.com/monitor-type"]; ok {
+		body += "&type=" + val 
+	}
```

Comment:
An else statement here will ensure that if this annotation is not provided, it defaults to type=1 because type is a mandatory option to pass when creating monitors in the API

Reply:
It should be fixed with this else